source/SemVerAnalyzer/Nuget/NugetClient.cs

```diff
@@ -59,5 +88,22 @@ static byte[] ReadAllBytes(Stream input)
 
 			return ms.ToArray();
 		}
+
+		class NugetPackageFeed {
+			[JsonPropertyName("resources")]
+			public IEnumerable<Resource> Resources { get; set; }
+		}
+		class Resource {
+			[JsonPropertyName("@id")]
+			public string Id { get; set; }
+			[JsonPropertyName("@type")]
+			public string Type { get; set; }
+		}
+		class VersionsFeed {
+			[JsonPropertyName("versions")]
+			public IEnumerable<string> Versions { get; set; }
+		}
+
 	}
+
```

Comment:
What's with the extra blank lines before the `}`s?

Reply:
This is public open source. I don't think it needs to be as strictly covered by our coding guidelines.